core/src/test/java/org/ehcache/util/EhcacheBasicTestUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.util;
+
+import org.ehcache.Ehcache;
+import org.ehcache.event.CacheEventListener;
+import org.ehcache.event.EventFiring;
+import org.ehcache.event.EventOrdering;
+import org.ehcache.event.EventType;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author
+ */
+public class EhcacheBasicTestUtil {
+
+    public static void registerCacheEventListener(Ehcache ehcache,CacheEventListener cacheEventListener){
```

Comment:
a Util class with a static method is a code smell, here a solution would be to  have this method in a parent class from which all your test classes inherit